plugins/GSdx/Renderers/OpenGL/GSRendererOGL.cpp

```diff
@@ -177,14 +177,14 @@ void GSRendererOGL::EmulateZbuffer()
 	const bool clamp_z = (uint32)(GSVector4i(m_vt.m_max.p).z) > max_z;
 
 	vs_cb.MaxDepth = GSVector2i(0xFFFFFFFF);
```

Comment:
Did you check if the same could apply also here ?

Reply:
We checked those, that all aligns fine on the vertex buffer